app/src/db/migrations/20200603145742_011-agriseafoodopscreening-data.js

```diff
@@ -0,0 +1,206 @@
+        config: {
+          template: 'confirmation-number-email.html',
+          from: EMAIL_FROM,
+          subject: `${FORM_NAME} Form Submission`,
+          title: `${FORM_NAME} Form Submission`,
+          priority: 'normal',
+          messageLinkText: 'Click to view your Submission',
+          messageLinkUrl: `https://comfort-dev.pathfinder.gov.bc.ca/app/${SLUG}/review`
+        },
+        createdBy: 'migration-006'
+      };
+      return knex(`${PREFIX}_settings`).insert(data);
+    })
+    .then(() => {
+      const data = {
+        name: 'confirmationEmail',
+        enabled: true,
+        config: {
+          template: 'confirmation-number-email.html',
+          from: EMAIL_FROM,
+          to: 'NR.CommonServiceShowcase@gov.bc.ca',
+          subject: `${FORM_NAME} Form Accepted`,
+          title: `${FORM_NAME} Form Accepted`,
+          priority: 'normal',
+          messageLinkText: `Please login to view the details of this ${FORM_NAME} Attestation`,
+          messageLinkUrl: `https://comfort-dev.pathfinder.gov.bc.ca/app/${SLUG}/admin/submission`
+        },
+        createdBy: CREATED_BY
+      };
+      return knex(`${PREFIX}_settings`).insert(data);
+    })
+    .then(() => {
+      const data = {
+        name: 'statusAssignmentEmail',
+        enabled: true,
+        config: {
+          template: 'status-assignment-email.html',
+          from: EMAIL_FROM,
+          subject: `${FORM_NAME} Assignment`,
+          title: `${FORM_NAME} Assignment`,
+          priority: 'normal',
+          message: 'The status of the above attestation has changed. You have been assigned to review this attestation.',
+          messageLinkText: `Please login to view the details of this ${FORM_NAME} Attestation`,
+          messageLinkUrl: `https://comfort-dev.pathfinder.gov.bc.ca/app/${SLUG}/admin/submission`
+        },
+        createdBy: CREATED_BY
+      };
+      return knex(`${PREFIX}_settings`).insert(data);
+    })
+    .then(() => {
+      const data = {
+        name: 'generateSubmissionPdf',
+        enabled: true,
+        config: {
+          template: 'generate-submission-pdf-001.html',
+          templateJson: 'generate-submission-pdf-001.json'
+        },
+        createdBy: CREATED_BY
+      };
+      return knex(`${PREFIX}_settings`).insert(data);
+    })
+    .then(() => {
+      const data = {
+        name: 'accessRequestedEmail',
+        enabled: true,
+        config: {
+          template: 'access-requested-email.html',
+          from: EMAIL_FROM,
+          to: 'NR.CommonServiceShowcase@gov.bc.ca',
+          subject: `${FORM_NAME} - Access Requested`,
+          title: 'Access Requested',
+          priority: 'normal',
+          message: `A request for access has been received for your ${FORM_NAME} administration page.`,
+          messageLinkText: 'Please Login to view the Access Request(s).',
+          messageLinkUrl: `https://comfort-dev.pathfinder.gov.bc.ca/app/${SLUG}/admin/team`
```

Comment:
and this will be prod

Reply:
We have to manually change the data in the db as it gets promoted.  This is a known step, so we update urls and update email addresses.  We have no multi-environment data migrations.  Although, maybe we could explore something where we read env vars for values from the settings table...?

Reply:
ok. good to know.
using .env files for each environment can work too i assume
i guess having it in a settings table would allow an admin to update it, that sounds like the best way

Reply:
hmmm.  yeah.  wonder if we can leverage env vars as we run the migration, then the db is populated with "correct" data that can be fully manipulated in the db... definitely something to explore.  this manual step of updating data is not my favourite.